services/ui-src/src/components/reports/ReportPageIntro.tsx

```diff
@@ -30,6 +31,9 @@ export const ReportPageIntro = ({
           />
         </Box>
       )}
+      {showIlosAlert && (
+        <Alert status={AlertTypes.WARNING} description={alert} />
+      )}
```

Comment:
Nit: Instead of making this alert ilos specific, could we just check for an alert in the verbiage to apply? Then we don't need to care about page type and we don't need to tie it into anything.

For example:
```
      {alert && (
        <Alert status={AlertTypes.WARNING} description={alert} />
      )}
```

If we went further and defined alert as an object type, we could give it a status to pass as well as a description. That way we never need to worry about the alert again.

Reply:
I changed the check! great call. I think for the Alert type, we can create a separate PR and do some more testing, I'll add a ticket for that 💯